Day-II-Calculator/calculator/src/components/ButtonComponents/ButtonContainer.js

```diff
@@ -0,0 +1,46 @@
+import React, {Fragment} from 'react';
+import './Button.css';
+import ActionButton from './ActionButton';
+import NumberButton from './NumberButton';
+
+function ButtonContainer(props) {
+    
+    function callback(value) {
+        props.onValueReceived(value);
+    }
+    
+    return (
+        <Fragment>
+            <div className="row">
+                <ActionButton value="clear" buttonWidth="wide clear" whenClicked={(value) => callback(value)} />
```

Comment:
Why add an extra step by passing the whenClicked `callback` function this way? Get rid of it by just passing the function (without invoking it inside of another):
```javascript
whenClicked={callback}
```

Reply:
The same goes for the rest of the places where you pass it in that way!